app/assets/stylesheets/views/_landing_page.scss

```diff
@@ -0,0 +1,19 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.landing-page-header {
+  // I've used the hex value for the background colour as it doesn't exist in the colour palette
+  background-color: #231F20;
+  width: 100%;
```

Comment:
Width isn't needed here?

Reply:
Quite right, think it was left over from me messing around with some things. Will remove